temp_examples/scenic-streetmap-overlay/scenic_plot.html

```diff
@@ -0,0 +1,398 @@
+              return {
+                fillColor: percToColour(feature.properties.scenic_rating, 2.16, 4.84),
+                weight: 1,
+                opacity: 0,
+                color: 'white',
+                dashArray: '1',
+                fillOpacity: .6
+              }; 
+            }
+          });
+          
+          var crimeLayer = L.geoJson(json, { // crime layer displaying mean monthly crime counts per LSOA
+            style: function (feature) {
+              return {
+                fillColor: getCrimeColour(feature.properties.mean_monthly_crime_count),
+                weight: 1,
+                opacity: 0,
+                color: 'white',
+                dashArray: '1',
+                fillOpacity: .4
+              }; 
+            }
+          });
+
+          var d3Layer = L.Class.extend({ // extend leaflet class to toggle on/off d3 layer w/ other leaflet layers
+            initialize: function() {
+              return;
+            },
+            onAdd: function() {
+              leafletMap.on('viewreset', mapmove); // remove points and redraw relevant subset as we move around map
+              redrawSubset(geoData.features); // draw initial susbet in starting position
+            },
+            onRemove: function() {
+              leafletMap.off('viewreset', mapmove);
+              d3.selectAll(".point").remove();
+            },
+          });
+          svgPointsLayer = new d3Layer(); 
+          
+          // switch between scenic rating per points vs mean per LSOA vs crime counts
+          var baseMaps = {
+            "Average Scenic Ratings per LSOA": sceneryLayer,
+            "Average Crime Count per Month": crimeLayer,
+            "Scenic Points": svgPointsLayer
+          };
+          L.control.layers(baseMaps, null, {position: 'bottomleft', collapsed: false}).addTo(leafletMap);
+          leafletMap.attributionControl.addAttribution('Scenic Data &copy; <a href="http://scenicornot.datasciencelab.co.uk/">Scenic-Or-Not</a>');
+        });
+
+        /**
+         * Then, add and scale info bars (scenic rating and indices of health, income and employment deprivation) 
+         * to show when particular LSOA is clicked on. Each variable has been scaled between 1 and 10.
+         */
+        
+        // custom info control
+        var info = L.control();
+        info.onAdd = function (map) {
+          this._div = L.DomUtil.create('div', 'info'); // create a div with a class "info"
+          this.update();
+          return this._div;
+        };
+
+        // method to update the control based on feature properties passed
+        info.update = function (props) {
+          var rollover_html=``;
+          if ((!props)&&(!now_showing)) {
+            rollover_html=`<h4>Click on any area <br />to see region info</h4>`;
+          } else if ((props)&&(!now_showing)) {
+            rollover_html+=`<h4>Show info for `+props.lsoa11cd+` (`+props.lsoa11nm+`)</h4>`;	
```

Comment:
When I mocked up that example quickly for you, I wasn't thinking too much about the wording. Perhaps it should read "Show data for XX"

It would also be good to have the Crime in this drop down. I can't re,eber if this is easy to do, but perhaps the "Show next" should also be after the data at the bottom of the box to top the data from jumping up and down. I hope that makes sense.